apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/SpansResourceTest.java

```diff
@@ -3384,6 +3390,17 @@ private void assertIgnoredFields(List<Span> actualSpans, List<Span> expectedSpan
                                     .build())
                     .isEqualTo(expectedFeedbackScores);
 
+            SoftAssertions.assertSoftly(softly -> {
```

Comment:
There's only one assertion executed in each test run, I don't see why we need a soft assertion here